app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductNutritionFactsFragment.java

```diff
@@ -754,17 +755,19 @@ private boolean isDataPer100() {
 
     @OnClick(R.id.btn_add_a_nutrient)
     void addNutrient() {
+        String[] nutrients = getResources().getStringArray(R.array.nutrients_array);
+        Arrays.sort(nutrients);
```

Comment:
This sort processes only unicode characters.

```
["Zorro", "Almidón", "Ácido esteárico"]  // before sort
["Almidón", "Zorro", "Ácido esteárico"]  // after sort
```
`Á` comes after  `Z`.

Need to write a custom sort for later.

Reply:
Now the sorting method ignore the accents.
```
["Ácido esteárico", "Almidón", "Zorro"]  // after sort
```

See the tests in `StringComparatorTest`.
I don't know if `StringComparator` is in the right package `utils`.